ci/install.sh

```diff
@@ -14,7 +14,11 @@ echo "Install neovim npm package"
 npm install -g neovim
 npm link neovim
 
-if [[ $CI_OS_NAME != osx ]]; then
+perl_found=$(perl -e 'if ( $^V gt 'v5.22.0' ) {print 1}')
```

Comment:
Checking for a random perl version here means we need to think about the required perl version needed by Neovim:Ext's transitive deps. 

Neovim::Ext needs to deal with this, not us.

Not interested in spending more time on this. What we need is (1) to report the issue to Neovim::Ext and (2) decide if we are going to bother having this in CI at all.

Reply:
fyi, this isn't a random version, see the error message here

- https://github.com/neovim/neovim/runs/6687689512?check_suite_focus=true#step:8:49

and the transitive requirement here

- https://metacpan.org/pod/Eval::Safe#:~:text=Eval%3A%3ASafe%20is%20a,t%20try%20to%20be%20safe.

>  (2) decide if we are going to bother having this in CI at all.

as I suggested earlier on Matrix, we are probably better off with testing the [go-client](https://github.com/neovim/go-client) or [rust-client](https://github.com/KillTheMule/nvim-rs) instead